solutions/core-landing-zone/README.md

```diff
@@ -1,6 +1,7 @@
 <!-- BEGINNING OF PRE-COMMIT-BLUEPRINT DOCS HOOK:TITLE -->
 # core-landing-zone
 
+TODO: Regenerate README
```

Comment:
the client-landing-zone readme will also need to be regenerated and will probably conflict with PR #400 (whichever is merged first)

Reply:
I will add a TODO